css/styles.css

```diff
@@ -0,0 +1,68 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.main-header {
+  display: flex;
+  flex-direction: row;
+  background: #3c3a39;
+  justify-content: space-around;
```

Comment:
- [ ] kindly use `justify-content: space-between; padding: 0 15px;` to make your toolbar match the design.